src/php/php.ini

```diff
@@ -720,7 +720,7 @@ user_dir =
 
 ; Directory in which the loadable extensions (modules) reside.
 ; http://php.net/extension-dir
-; extension_dir = "./"
+extension_dir = "${SNAP}/lib/php/extensions/no-debug-non-zts-20151012"
```

Comment:
`no-debug-non-zts-20151012` feels like a build artifact. Is it possible to use `${SNAP}/lib/php/extensions/` ?